### DISCUSSION TITLE
Refer another Jslt file inside Jslt

### DISCUSSION THREAD

**Discussion by MailNSrinivasan** [schibsted/jslt#295] [Q&A] [answered]
Hi 

Thanks for this wonderful tool!

I have a question to see do we have a feature to refer to another jslt file, not as import.  I am building jslt file which has a field "details" which is a json object which has a huge set of fields inside. If I can reference another jslt file inside the main jslt I can separate those two into different files and have clean files instead of me having 100+ fields in one jslt. 

Thanks
Srini

**Comment by larsga** [comment, marked as answer]
Well ...  I would say this is exactly what `import` is supposed to do for you. Why not use `import`?

**Reply by MailNSrinivasan** [reply]
ohh I read through the document: https://github.com/schibsted/jslt/blob/master/ideas/import.md


import "transforms/pulse-identified.jstl2" as identified
import "transforms/pulse-anonymized.jstl2" as anonymized

if (.actor."spt:userId" and not(ends-with(":null", .actor."spt:userId")))
  identified(.)
else
  anonymized(.)


I can use like identified(.) to bring in the transformed data. 
I will try that. Thanks
